product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -5,81 +5,70 @@ redirects:
     - 01_user_access
 ---
 
-BigAnimal uses role-based access controls to grant users access to different parts of the application using organization- and project-level roles. Roles are sets of permissions. You use roles to manage permissions assigned to users.
+BigAnimal uses role-based access controls to grant users access to different parts of the application using organization-level and project-level roles. Roles are sets of permissions. Roles are used to manage permissions assigned to users.
 
-Each subscribed customer has a unique *organization*. Each organization can have one or more projects. Projects within an organization segregate workflows by providing secure and isolated environments. An organization's database clusters are deployed and managed in the customer's cloud account within a project. This approach ensures complete segregation of customer data between organizations and projects within organizations.
+Each customer has a unique BigAnimal organization. Each organization will have at least one project by default; more projects can be created. An organization's database clusters are deployed and managed in the customer's cloud account within a project. Projects within an organization separate workflows to provide secure and isolated environments to ensure complete separation of customer data between different projects within an organization.
```

Comment:
We are using a lot of passive voice in the first two paragraphs now, which is not ideal. Maybe we can work together, Natalia, to see if we can make this more active without changing the meaning. Or, we can leave the passive voice for now.

Reply:
agree, let's get rid of the passive voice.